dev/tools/pre-commit

```diff
@@ -7,10 +7,18 @@ set -e
 
 dev/tools/check-overlays.sh
 
+# Can we check and fix formatting?
+ocamlformat=$(command -v ocamlformat || echo true)
+if $ocamlformat &> /dev/null
```

Comment:
I'd do `[[ $ocamlformat = true ]]` or similar (feels easier to understand what's going on)